apps/bettafish/src/app/pages/tags/tag-page.component.html

```diff
@@ -1,49 +1,56 @@
-    </div>
-</dragonfish-topbar>
-
-<ng-container *ngIf="tagsQuery.loading$ | async else notLoading">
-    <div class="flex flex-row items-center justify-center h-72">
-        <mat-spinner></mat-spinner>
-    </div>
-</ng-container>
+<dragonfish-topbar></dragonfish-topbar>
+<ng-scrollbar>
+    <ng-container *ngIf="tagsQuery.loading$ | async else notLoading">
+        <div class="flex flex-row items-center justify-center h-72">
+            <mat-spinner></mat-spinner>
+        </div>
+    </ng-container>
 
-<ng-template #notLoading>
-    <ng-container *ngIf="tagsQuery.all$ | async as tags">
-        <ng-container *ngIf="tags.length === 0; else notEmpty">
-            <div class="empty">
-                <h3>Looks like there something missing here...</h3>
-                <p>If you think this is an error, contact your system administrator.</p>
-            </div>
-        </ng-container>
-        <ng-template #notEmpty>
-            <div class="w-8/12 mx-auto my-8">
-                <mat-accordion>
-                    <ng-container *ngFor="let tag of tags">
-                        <mat-expansion-panel>
-                            <mat-expansion-panel-header>
-                                <mat-panel-title>
-                                    {{ tag.name }}
-                                </mat-panel-title>
-                                <mat-panel-description>
-                                    {{ tag.desc }}
-                                </mat-panel-description>
-                            </mat-expansion-panel-header>
-                            <div class="my-2"><!--spacer--></div>
-                            <ul>
-                                <ng-container *ngFor="let child of tag.children">
-                                    <li class="py-2 px-1 border-t" style="border-color: var(--borders);">
-                                        <dragonfish-child-tag-item [parentId]="tag._id" [tag]="child"></dragonfish-child-tag-item>
-                                    </li>
-                                </ng-container>
-                            </ul>
-                        </mat-expansion-panel>
+    <ng-template #notLoading>
+        <ng-container *ngIf="tagsQuery.all$ | async as tags">
+            <ng-container *ngIf="tags.length === 0; else notEmpty">
+                <div class="empty">
+                    <h3>Looks like there something missing here...</h3>
+                    <p>If you think this is an error, contact your system administrator.</p>
+                </div>
+            </ng-container>
+            <ng-template #notEmpty>
+                <div class="w-11/12 mx-auto my-6">
+                    <h3>
+                        <ng-container *ngIf="tags[0].parent">
+                            <a
+                                [routerLink]="['/tag', tags[0].parent._id, tags[0].parent.name | slugify]"
+                                [innerHtml]="tags[0].parent.name | safeHtml"
```

Comment:
nitpick: Angular's recommend styles tend to avoid `innerHtml` in favor of `{{ interpolation }}`. I don't think we have a precedent in the codebase, but as a _personal_ preference, I find interpolation a bit more readable, i.e.
```html
<a [routerLink]="['/tag', tags[0].parent._id, tags[0].parent.name | slugify]">
  {{ tags[0].parent.name | safeHtml }}
</a>
```

As an aside, you shouldn't need to run a tag name through `safeHtml`, as tag names shouldn't contain HTML.

...that said, do we have anything preventing a malicious moderator from creating a tag name with HTML tags in it?

Reply:
That's how it was before, and it didn't process the HTML correctly